locator/topology.cc

```diff
@@ -105,7 +101,7 @@ topology::topology(topology&& o) noexcept
 }
 
 future<topology> topology::clone_gently() const {
-    topology ret;
+    topology ret(_cfg);
```

Comment:
This constructor has an `add_node` call in it, unlike the previously used default constructor.
Won't this potentially cause `add_node` to be called twice for the same node - by the constructor and later by the `for` loop below?

Reply:
Hm, but I see you remove the `add_node` call in the constructor in a later commit.